### ISSUE TITLE
NTR: Heptacarboxylporphyrin high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8302]
New term request Heptacarboxylporphyrin high in blood (elthat_121124191930): Porphyrins–fluorescence emission peak high

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000698
- label: Elevated circulating heptacarboxylporphyrin concentration
- parents: [Elevated circulating porphyrin concentration (HP:0034606)](https://hpo.jax.org/app/browse/term/HP:0034606)
- pmid: PMID:11929047
- def: The concentration of heptacarboxylporphyrin in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8302
- synonyms:
